### PR TITLE
Fixes #15095: Missing log on hooks for generation

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/15095

### CODE REVIEW

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PolicyWriterService.scala [hunk 1]
```diff
@@ -384,10 +386,13 @@ class PolicyWriterServiceImpl(
                                                              , ("RUDDER_POLICIES_DIRECTORY", agentNodeConfig.paths.baseFolder)
                                                            )
                                         , systemEnv
+                                        , 60*1000 // warn if a hook took more than a minute
                             )
                             HooksLogger.trace(s"Run post-generation post-move hooks for node '${nodeId}' in ${System.currentTimeMillis - timeHooks} ms")
                             res
                           }
+      postMvHooksTime2   = System.currentTimeMillis
+      _                  = policyLogger.debug(s"Hooks for policy-generation-node-ready exectuted in ${postMvHooksTime2 - movedPromisesTime2} ms")
```
Comment (amousset) [inline webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PolicyWriterService.scala]: `executed`

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
PR rebased

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**